examples/mcu-board-support/stm32h735g.rs

```diff
@@ -54,10 +54,7 @@ impl slint::platform::Platform for StmBackend {
     fn create_window(&self) -> Rc<dyn slint::platform::PlatformWindow> {
         let window = Rc::new_cyclic(|self_weak: &Weak<StmWindow>| StmWindow {
             window: slint::Window::new(self_weak.clone()),
-            renderer: swrenderer::SoftwareRenderer::new(
-                swrenderer::DirtyTracking::DoubleBuffer,
-                self_weak.clone(),
-            ),
+            renderer: swrenderer::SoftwareRenderer::new(self_weak.clone()),
```

Comment:
Ah, suggestion: In the swrenderer module we could also define two convenience type aliases:

```rust
type DoubleBufferSoftwareRenderer = SoftwareRenderer<2>;
type SingleBufferSoftwareRenderer = SoftwareRenderer<1>;
```

Reply:
True, it could.   But I don't think we'll need this because it will be hidden in the MinimalWindow anyway.